src/ta/mod.rs

```diff
@@ -50,7 +50,7 @@ mod tests {
     #[test]
     fn init_ta() {
         test::test_in_memory(|storage_uri| {
-            let cleanup = test::init_logging();
+            let _ = stderrlog::new().verbosity(5).init();
```

Comment:
Where does 5 come from?

Reply:
I suppose from me misreading the documentation and thinking “higher than 4” is for trace rather than “4 or higher.” Looking at the source, they same to use `LevelFilter` internally, so I’ll change it to that.